content/dart/concepts/list/terms/join/join.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'join' 
+Description: 'The join() method concatenates (combines) list elements as a string including various separators between them dependent on a user's need.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+Tags:
+  - 'Concatenation'
+  - 'Dart'
+  - 'Lists'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+The **`join()`** method concatenates (*combines*) list elements as a string including various separators between them dependent on a user's need.
+ 
+
+## Syntax
+
+```pseudo
+list-name.join(separator); // the raw syntax
+
+chores.join(" + "); // a real example
+```
+
+The method refers to its object, in the below case, to the 'chores' variable, which holds list (or array) elements. Inside the brackets '( )' there's a parameter - a separator containing any sign/-s wrapped with single/double quotes. The main role of a separator is to separate elements from one another - allowing for better readability and additional or new meaning to the text. Spaces added to a separator are nonobligatory.
+
+
+## Example 1
+
+The following block codes are showing an instance of removed spaces between elements, however a default `,` remains:
+
+```dart
+List<String> tought_to_read = ["A", "bit", "hard", "to", "read", "innit", "?"];
+
+String try_to_read = tought_to_read.join(); // Spaces between elements are removed.
+print(try_to_read); 

```

Comment:
The whole code block should be executable, you need to wrap the code inside the main function (Do this for every code block).

Reply:
All lists and variables are global ones and they don't need another function, except print() here, to be executed.